modules/drivers/snowflake/test/metabase/driver/snowflake_test.clj

```diff
@@ -87,13 +87,28 @@
                  :engine :snowflake
                  :private-key-creator-id 3
                  :user "SNOWFLAKE_DEVELOPER"
-                 :private-key-created-at "2024-01-05T19:10:30.861839Z"}]
-    (testing "Database name is quoted iff quoting is requested (#27856)"
+                 :private-key-created-at "2024-01-05T19:10:30.861839Z"
+                 :host ""}]
+    (testing "Database name is quoted if quoting is requested (#27856)"
       (are [quote? result] (=? {:db result}
                                (let [details (assoc details :quote-db-name quote?)]
                                  (sql-jdbc.conn/connection-details->spec :snowflake details)))
         true "\"v3_sample-dataset\""
-        false "v3_sample-dataset"))))
+        false "v3_sample-dataset"))
+    (testing "Subname is replaced if hostname is provided (#22133)"
+      (are [use-account-name alternative-host expected-subname] (=? expected-subname
+                               (:subname (let [details (-> details
+                                                           (assoc :host alternative-host)
+                                                           (assoc :use-account-name use-account-name))]
+                                 (sql-jdbc.conn/connection-details->spec :snowflake details))))
```

Comment:
Please fix the indentation.

Reply:
@metamben the linter doesn't input any issue. I'm not too familiar with closure, if there a script to check/fix indentation on the project?

Reply:
Never mind then, we'll fix it.